.github/workflows/build-wheels.yml

```
@@ -0,0 +1,44 @@
+name: Builds
+
+on:
+  push:
+    branches:
+    - master
+  pull_request:
+    branches:
+    - master
+  # Run weekly at 1:23 UTC
+  schedule:
+  - cron: '23 1 * * 0'
+  workflow_dispatch:
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository and submodules
+        uses: actions/checkout@v4
+        with:
+          submodules: recursive
+
+      - name: Set up Python
+        uses: actions/setup-python@v4
```

```suggestion
        uses: actions/setup-python@v5
```

Not your fault, but you've been patiently waiting for review so long that there's a new version out. :sweat_smile: